src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/listeners/HMPPSAuditListenerTest.kt

```diff
@@ -1,23 +1,18 @@
 package uk.gov.justice.digital.hmpps.hmppsauditapi.listeners
 
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
 import com.nhaarman.mockitokotlin2.check
+import com.nhaarman.mockitokotlin2.mock
 import com.nhaarman.mockitokotlin2.verify
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
-import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.boot.test.mock.mockito.MockBean
-import org.springframework.test.context.ActiveProfiles
 import uk.gov.justice.digital.hmpps.hmppsauditapi.services.AuditService
 
-@SpringBootTest
-@ActiveProfiles("test")
 internal class HMPPSAuditListenerTest {
-  @MockBean
-  private lateinit var auditService: AuditService
-
-  @Autowired
-  private lateinit var listener: HMPPSAuditListener
+  private val auditService: AuditService = mock()
+  private val listener: HMPPSAuditListener =
+    HMPPSAuditListener(auditService = auditService, ObjectMapper().registerModule(JavaTimeModule()))
```

Comment:
I was just checking to see how the object mapper is configured in the main application - is there an issue with indentation in the jackson config in `application.yml`?  If so it's a shame that wasn't caught by a test...

Reply:
Have reverted to integrationtest to allow spring to inject the objectmapper directly - this is a better test to use the same config of the mapper when running the application